packages/components/src/panel/body.native.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+export class PanelBody extends Component {
+	constructor( ) {
+		super( ...arguments );
+		this.state = {};
+	}
+
+	render() {
+		const { children } = this.props;
+		return (
+			<View >
+				{ children }
+			</View>
+		);
+	}
+}
+
+export default PanelBody;
```

Comment:
Minor suggestion, but if this has no custom behavior other than rendering a View, we could just do:

```jsx
/**
  * External dependencies
  */
import { View } from 'react-native';

export default View;
```

Reply:
It's because I wrapped all content inside `Inspector Controls` in `PanelBody` 

At this moment I also know that there will be additional work referring to PanelBody stars in few days so would you mind if we can keep it as it is ?

Reply:
I think it's ok to keep it if it's going to get filled in soon